stock_shipment_management/wizard/create_shipment.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Yannick Vaucher
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+from openerp import models, api, fields, exceptions, _
+
+
+class ShipmentPlanCreator(models.TransientModel):
+
+    """Create a shipment plan from stock.picking or stock move.
+    This will take all related stock move from the selected picking
+    and put them in the shipment plan."""
+
+    _name = 'shipment.plan.creator'
+    _description = 'Shipment Plan Creator'
+
+    shipment_id = fields.Many2one(
+        'shipment.plan', 'Shipment',
+        domain="[('state', '=', 'draft'),"
+               " ('from_address_id', '=', from_address_id),"
+               " ('to_address_id', '=', to_address_id)]",
+        help="Shipment to which moves will be added.\nOnly shipment in draft "
+             "can be extended.",
+    )
+
+    move_ids = fields.One2many(
+        compute=lambda rec: True,
```

Comment:
This was already in version on stock-logistics-workflow

I created an issue to solve this later https://github.com/OCA/stock-logistics-transport/issues/11